src/scales/scale.logarithmic.js

```diff
@@ -94,30 +85,19 @@ class LogarithmicScale extends Scale {
 		let max = me.max;
 
 		if (min === max) {
-			if (min !== 0 && min !== null) {
-				min = Math.pow(10, Math.floor(log10(min)) - 1);
-				max = Math.pow(10, Math.floor(log10(max)) + 1);
-			} else {
+			if (min <= 0) { // includes null
```

Comment:
What if `min !== max` and `min <= 0` and `max <= 0`? Will we / should we use the defaults in that case?

Reply:
`<= 0` will be parsed to `NaN`. If parsing is disabled though, I think that would be the user's responsibility to make sure the data is valid.

Reply:
Oh, and `determineDataLimits()`: 
```js
		me.min = isFinite(min) ? Math.max(0, min) : null;
		me.max = isFinite(max) ? Math.max(0, max) : null;
```